tests/Unit/database/migrations/CreateOrdersTableTest.php

```
@@ -66,7 +66,28 @@ public function it_checks_if_orders_table_has_all_required_columns(): void
     public function it_checks_if_orders_table_column_types(): void
     {
         // Test UUID columns
-        $this->assertEquals('uuid', Schema::getColumnType('orders', 'id'));
+        if (config('database.default') === 'pgsql') {
+            // PostgreSQL returns 'uuid' for UUID columns
+            $this->assertEquals('uuid', Schema::getColumnType('orders', 'id'));
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Schema' in method 'it_checks_if_orders_table_column_types'. (Clean Code Rules)

(StaticAccess)